src/components/Layout/index.tsx

```diff
@@ -46,6 +46,12 @@ const Layout: FC<Props> = ({ children }) => {
                         router.pathname === '/mine'
                     )}
                     |
+                    {navBarContent(
```

Comment:
why are we manually adding this in the JSX ?

Reply:
this is straight forward you dont manually need to call this function again and again make an array and map over with your function

Reply:
fixed